### DISCUSSION TITLE
Why Eureka Effect is restricted to use with teleporters only on bosses?

### DISCUSSION THREAD

**Discussion by naydef** [Batfoxkid/Freak-Fortress-2-Rewrite#201] [Q&A] [answered]
https://github.com/Batfoxkid/Freak-Fortress-2-Rewrite/blob/0feb9dae5e809cdf28be2763383db7a274f90ee9/addons/sourcemod/scripting/freak_fortress_2/commands.sp#L327

I wonder for the particular purpose of this restriction. Broken maps? Delay prevention?

I guess this code can be made switchable with convar.

**Comment by Batfoxkid** [comment, marked as answer]
The intention was to indeed reduced the amount of stalling ability by being able to teleport to two different spots instead of one.
